features/features.go

```diff
@@ -599,6 +599,7 @@ var (
 					contactPerson("jspeed").
 					productScope(ocpSpecific).
 					enhancementPR(legacyFeatureGateWithoutEnhancement).
+					enableIn(configv1.TechPreviewNoUpgrade).
```

Comment:
Anything enabled in TPNU must also be enabled in DPNU
```suggestion
					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```